site/blocks/StickyNav.tsx

```diff
@@ -147,19 +148,21 @@ class StickyNav extends React.Component<
 
     componentDidMount() {
         this.filterValidLinks()
-        // Web fonts change the position of elements
-        // so we set the active heading only once we know where everything is.
-        // The alternative is to call setHeadingPositions as soon as the component mounts,
-        // but this can make currentHeadingIndex jump around.
-        window.addEventListener("load", this.handleResize)
         window.addEventListener("scroll", this.handleScroll)
```

Comment:
One more thing here that just comes to mind now:
You can use _passive_ scroll event listeners here, so we don't even run the chance of it leading to scroll jank: https://web.dev/uses-passive-event-listeners/